showcase/src/app/demo/collapsible/collapsible.component.html

```diff
@@ -0,0 +1,80 @@
+<example-viewer [extensions]="['HTML', 'CSS', 'TS']" filesPath="collapsible/collapsible-default" exampleTitle="SHOWCASE.DEMO.COLLAPSIBLE.DEFAULT_ICON">
+	<stark-collapsible 
+			htmlId="id0" 
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_0" 
+			[(isExpanded)]="collapsed[0]"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<button color="primary" class="collapsible-demo-button" (click)="toggleCollapsible(0)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML', 'CSS', 'TS']" filesPath="collapsible/collapsible-custom1" exampleTitle="SHOWCASE.DEMO.COLLAPSIBLE.CUSTOM_ICON">
+	<stark-collapsible
+			htmlId="id1"
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_1"
+			[(isExpanded)]="collapsed[1]"
+			icon="chevron-down"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<button color="primary" class="collapsible-demo-button" (click)="toggleCollapsible(1)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+	<hr class="collapsible-demo-line">
+	<stark-collapsible
+			htmlId="id2"
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_1"
+			[(isExpanded)]="collapsed[2]"
+			icon="information"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<button color="primary" class="collapsible-demo-button" (click)="toggleCollapsible(2)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+	<hr class="collapsible-demo-line">
+	<stark-collapsible
+			htmlId="id3"
+			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_1"
+			[(isExpanded)]="collapsed[3]"
+			icon="account-circle"
+	>
+		{{ 'SHOWCASE.DEMO.COLLAPSIBLE.HTML_CONTENT' | translate }}
+	</stark-collapsible>
+	<button color="primary" class="collapsible-demo-button" (click)="toggleCollapsible(3)" mat-raised-button>{{ 'SHOWCASE.DEMO.COLLAPSIBLE.TOGGLE_COLLAPSIBLE' | translate }}</button>
+</example-viewer>
+
+<example-viewer [extensions]="['HTML', 'CSS', 'TS']" filesPath="collapsible/collapsible-custom2" exampleTitle="SHOWCASE.DEMO.COLLAPSIBLE.CUSTOM_OPTIONS">
```

Comment:
This example can be removed since we won't implement the new "options" for the moment